regolith/validators.py

```diff
@@ -109,4 +113,35 @@ def ensure_email(email):
     'databases': (always_false, ensure_databases),
     'stores': (always_false, ensure_stores),
     'email': (always_false, ensure_email),
-    }
+}
+
+
+def pop_description(schema):
+    """Pop description off the schema since Cerberus doesn't support it"""
+    if isinstance(schema, dict):
+        schema.pop('description', None)
+        for v in schema.values():
+            pop_description(v)
+
+
+def validate(db, record):
+    """Validate a record for a given db
+    
+    Parameters
+    ----------
+    db : str
+        The name of the db in question
+    record : dict
+        The record to be validated
+
+    Returns
+    -------
+    bool:
+        True is valid
+
+    """
+    from regolith.schemas import SCHEMAS
+    schema = SCHEMAS[db]
+    pop_description(schema)
+    v = Validator(schema, allow_unknown=True)
```

Comment:
I don't think we want this, in general. Are you doing this only for the citations entry?  Maybe there is a way to add allow unknow for only certain schema.

Reply:
We can put it into each schema level as needed. http://docs.python-cerberus.org/en/stable/usage.html#allowing-the-unknown

Reply:
Great! Do that then :)